README.md

```diff
@@ -112,7 +118,7 @@ Let's replace the main `div` with `Grommet`.
 
 ```diff
 - <div className="App">
-+ <Grommet plain>
++ <Grommet>
```

Comment:
should we do `full="min"` here?

Reply:
Im also a little confused why we tell them to put `plain` and then tell them to take it out 

I know this was previous work just putting this out there

Reply:
`plain` is no longer included. I also added `full` to Grommet.